modules/monitoring-modifying-retention-time-and-size-for-prometheus-metrics-data.adoc

```diff
@@ -6,7 +6,12 @@
 [id="modifying-retention-time-and-size-for-prometheus-metrics-data_{context}"]
 = Modifying the retention time and size for Prometheus metrics data
 
-By default, Prometheus automatically retains metrics data for 15 days. You can modify the retention time for
+By default, Prometheus automatically retains metrics data for the following durations:
```

Comment:
(nit) do we need "automatically"?

Reply:
Good point, I agree, we already say "by default"  the word "automatically" here seems redundant, thank you!